### PR TITLE
Impement func repotagFromRPMFile to compute image metadata file

### PR DESCRIPTION
**Pull request by mjura**
Work In Progress and still wasn't tested yet

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
still needs some `gofmt` love :-) ...maybe we should also configure travis to do that check as well, just to be sure

**Comment by mjura** [comment]
Metadata will be generated by this change https://github.com/SUSE/containment-rpm-docker/pull/6

**Comment by mjura** [comment]
I've tested this change and works fine

File: feeder.go [hunk 1]
```diff
@@ -101,24 +132,41 @@ func (f *Feeder) imagesToImport(path string) (map[string]string, error) {
...
 	return images, nil
 }
 
 // Compute the repotag (`<name>:<tag>`) starting from the name of the tar.xz
 // file shipped by RPM
-func repotagFromRPMFile(file string) string {
-	// TODO: Implent code once the schema of the filename is defined
-	return file
+func repotagFromRPMFile(file string) (string, string, error) {
+
```
Comment (mssola) [inline feeder.go]: Remove empty line

File: feeder.go [hunk 2]
```diff
@@ -36,6 +38,35 @@ type FeederLoadResponse struct {
...
+For example:
+{
+  "image": {
+    "name": "opensuse/salt-api",
+    "tags": [ "13", "13.0.1", "latest" ],
+    "file": "salt-api-2017.03-docker-images.x86_64.tar.xz"
+  }
+}
+*/
+// MatadataType struct to handle JSON schema
```
Comment (mssola) [inline feeder.go]: Typo: "MatadataType" -> "MetadataType"

**Comment by jordimassaguerpla** [comment]
lgtm

**Pull request merged**